.github/workflows/release.yaml

```diff
@@ -0,0 +1,243 @@
+name: Publish release
+
+env:
+  GRADLE_OPTS: '-Dorg.gradle.daemon=false -Dkotlin.compiler.execution.strategy=in-process'
+  JAVA_VERSION: 11
+  NODEJS_VERSION: 14
+  PROTOBUF_VERSION: 3.10.1
+  BINTRAY_API_USER: ${{ secrets.BINTRAY_API_USER }}
+  BINTRAY_API_KEY: ${{ secrets.BINTRAY_API_KEY }}
+
+on:
+  release:
+    types:
+      - created
+
+jobs:
```

Comment:
All jobs are identical to the ones from `pull-request.yaml`, only the `publish` job got added.
https://github.com/streem/pbandk/blob/master/.github/workflows/pull-request.yaml

Reply:
The duplication is really unfortunate. But I know GitHub Actions doesn't make it easy to share jobs across workflows 😞 We can figure out some way to refactor this in the future so the jobs aren't duplicated, but this is fine for now.

Perhaps add a comment here that all jobs other than `publish` should be kept in sync with their counterparts in `pull-request.yaml`? And likewise add a comment in `pull-request.yaml` to keep the jobs in sync with `release.yaml`.